examples/nlp/language_modeling/conf/megatron_music_gpt_config.yaml

```diff
@@ -0,0 +1,203 @@
+name: megatron_music_gpt
```

Comment:
what's the difference between this conf vs megatron_gpt_config.yaml?
If it is just tokenizer difference, there is no need of new config file. just overwrite the tokenizer part.